lib/core/repository/cms/attribute_builder.rb

```diff
@@ -12,23 +12,21 @@ def self.build(response)
       end
 
       def attributes
-        attributes = response.body
+        attrs = response.body
```

Comment:
@tomas-stefano I did something like this and @norm had other ideas. I can see why both your ideas are viable. I need a second opinion on this. I am torn between whether to overload the method name or not. 

@jriga wdyt, please?